packages/fuels/src/cli/commands/deploy/proxy/contract/Forc.lock

```diff
@@ -0,0 +1,30 @@
+[[package]]
+name = "core"
+source = "path+from-root-C485BFF5467B6B6C"
+
+[[package]]
+name = "src14_owned_proxy"
+source = "member"
+dependencies = [
+    "standards",
+    "std",
+    "sway_libs",
+]
+
+[[package]]
+name = "standards"
+source = "git+https://github.com/FuelLabs/sway-standards?tag=v0.6.0#65e09f95ea8b9476b171a66c8a47108f352fa32c"
```

Comment:
@K1-R1 @bitzoic Should we use `0.6.1`?

> cc @danielbate

Reply:
Yes, the latest please.

Reply:
TLDR: 0.6.0 is correct here

The proxy contract here should be identical to https://github.com/FuelLabs/sway-standard-implementations/tree/master/src14/owned_proxy , as this is the latest audited proxy. This uses v0.6.0.

Forc deploy also uses this audited proxy; but rather than have the actual contract it uses a pre-build (from the sway-standard-implementations repo): https://github.com/FuelLabs/sway/tree/master/forc-plugins/forc-client/proxy_abi. If possible in this PR this would be preferable, as the binaries can be more easily verified and it won't be necessary to mirror any changes that may occur.

Reply:
Thanks @K1-R1. Great insights!

But what exactly did you mean with?
 1. as the binaries can be more easily verified
 1. it won't be necessary to mirror any changes that may occur

Reply:
1. I just mean that it's important to be able to easily verify that the proxy contract being deployed is the audited proxy, and by using the pre-built files directly from the sway-standard-implementations repo this is guaranteed.
2. If there audited proxy ever changed (not expected but possible in unforeseen events), it would only be necessary to update the pre-built files, instead of having to update a Sway contract.

I haven't had a chance to look deeply at this PR just yet; but another advantage of using the pre-build solution is that the proxy doesn't need to be compiled again when deploying new proxies

Reply:
# Binaries in use by `forc`

Ok, so should I use the same pre-built binaries that `forc` [is using](https://github.com/FuelLabs/sway/tree/master/forc-plugins/forc-client/proxy_abi)?
 - `174f5ed9c79c23a6aaf5db906fe27ecdb29c22eb`
 - https://github.com/FuelLabs/sway-standard-implementations/pull/19

# Binaries on `master`

Note that `forc` is not aligned with the `master` branch, which is at:
 - `a3eb2a6b093d349a84f379920df7eee9ad1ad5d9`
 - https://github.com/FuelLabs/sway-standard-implementations/pull/20

# Question

@K1-R1 Can you please confirm which commit I should use?
I suppose only docs changed between the two:
 - https://github.com/FuelLabs/sway-standard-implementations/compare/174f5ed9c79c23a6aaf5db906fe27ecdb29c22eb...a3eb2a6b093d349a84f379920df7eee9ad1ad5d9

Reply:
Let's keep parity across the board and use the same commit that `forc` is using: commit hash 174f5ed9c79c23a6aaf5db906fe27ecdb29c22eb .
As you mentioned there have been no changes to the contract itself so `forc` is still using the latest version of the contract.